fenix/app/src/main/java/org/mozilla/fenix/compose/ext/Int.kt

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose.ext
+
+import androidx.compose.ui.text.intl.Locale
+import java.text.NumberFormat
+import java.util.Locale as JavaLocale
+
+/**
+ * Returns a localized string representation of the value.
+ */
+fun Int.toLocaleString(): String =
```

Comment:
I decided to create a helper function for this.

I needed to use `Locale.current.language` in order to get the Compose previews working. For whatever reason, Java's `Locale.default()` does not work correctly in previews, causing the Locale preview parameter to be ignored when formatting numbers.